File: WebSocketChat.Frontend/src/app/shared/models/chat/messages/room-removal-message.ts
import { ChatRoom } from './../chat-room';
import { MessageIdentifier, Message } from './../message';
import { MessageType } from "app/shared/models/chat/message-type";


export class RoomRemovalMessage{

   messageType : MessageType = MessageType.RoomRemovalMessage;

   /**The removed room's identifier */
   roomIdentifier: number;
}

File: WebSocketChat.Server/WebSocketChat.Server/Models/Chat/Messages/RoomRemovalMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebSocketChat.Server.Models.Chat.Messages
{
    public class RoomRemovalMessage : Message
    {
        public RoomRemovalMessage()
        {
            MessageType = MessageType.RoomRemovalMessage;
        }

        /// <summary>
        /// Room identifier of the removed room
        /// </summary>
        public int RoomIdentifier { get; set; }
    }
}
